Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Header.css';
+<script src="moment.js"></script>
```

Comment:
Nice job going after the stretch.  for Moment, you can install it with yarn or npm, and then import it at the top of your file to use it.